### PR TITLE
Add s390x CI job for s390x

### PR DESCRIPTION
**Pull request by adevir91**

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 1]
```diff
@@ -162,6 +162,215 @@ periodics:
...
+"[sig-builds][Feature:Builds] build can reference a cluster service  with a build being created from new-build should be able to run a build that references a cluster service [Suite:openshift/conformance/parallel]"
+"[sig-builds][Feature:Builds] build have source revision metadata  started build should contain source revision information [Suite:openshift/conformance/parallel]"
+"[sig-builds][Feature:Builds] build with empty source  started build should build even with an empty source in build config [Suite:openshift/conformance/parallel]"
+"[sig-builds][Feature:Builds] build without output image  building from templates should create an image from a S2i template without an output image reference defined [Suite:openshift/conformance/parallel]"
+"[sig-builds][Feature:Builds] custom build with buildah  being created from new-build should complete build with custom builder image [Suite:openshift/conformance/parallel]"
+"[sig-builds][Feature:Builds] result image should have proper labels set  S2I build from a template should create a image from "test-s2i-build.json" template with proper Docker labels [Suite:openshift/conformance/parallel]"
```
Comment (r4f4) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:296]: escape the double quotes here

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 2]
```diff
@@ -162,6 +162,215 @@ periodics:
...
+  cluster: api.ci
+  cron: 0 4,16 * * *
+  decorate: true
+  labels:
+    dptp.openshift.io/ssh-bastion: "true"
+    job-env: remote-libvirt
+    job-release: "4.7"
+    job-test: e2e
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-origin-installer-e2e-remote-libvirt-s390x-4.7
```
Comment (r4f4) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:175]: The jobs need to be ordered by name. Move this job to the appropriate place in the file.

**Review by r4f4 — CHANGES_REQUESTED** [review]
Make sure you have the option to show trailing whitespaces set in your text editor. Some lines have trailing spaces at the end.

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 3]
```diff
@@ -755,6 +755,215 @@ periodics:
...
+            "[sig-builds][Feature:Builds] build have source revision metadata  started build should contain source revision information [Suite:openshift/conformance/parallel]"\
+            \"[sig-builds][Feature:Builds] build with empty source  started build should build even with an empty source in build config [Suite:openshift/conformance/parallel]"\
+            \"[sig-builds][Feature:Builds] build without output image  building from templates should create an image from a S2i template without an output image reference defined [Suite:openshift/conformance/parallel]"\
+            \"[sig-builds][Feature:Builds] custom build with buildah  being created from new-build should complete build with custom builder image [Suite:openshift/conformance/parallel]"\
+            \"[sig-builds][Feature:Builds] result image should have proper labels set  S2I build from a template should create a image from "test-s2i-build.json" template with proper Docker labels [Suite:openshift/conformance/parallel]"\
```
Comment (r4f4) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:889]: No the beginning quotes, the one surrounding `"test-s2i-build.json"`

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 4]
```diff
@@ -755,6 +755,215 @@ periodics:
...
+            name: cluster-secrets-metal
+    - configMap:
+        name: prow-job-cluster-launch-installer-remote-libvirt-e2e
+      name: job-definition
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: release-pull-secret
+      secret:
+        secretName: ci-pull-credentials
```
Comment (r4f4) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:966]: there are trailing spaces at the end of this line

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 5]
```diff
@@ -755,6 +755,215 @@ periodics:
...
+            "[sig-builds][Feature:Builds] build have source revision metadata  started build should contain source revision information [Suite:openshift/conformance/parallel]"
+            "[sig-builds][Feature:Builds] build with empty source  started build should build even with an empty source in build config [Suite:openshift/conformance/parallel]"
+            "[sig-builds][Feature:Builds] build without output image  building from templates should create an image from a S2i template without an output image reference defined [Suite:openshift/conformance/parallel]"
+            "[sig-builds][Feature:Builds] custom build with buildah  being created from new-build should complete build with custom builder image [Suite:openshift/conformance/parallel]"
+            "[sig-builds][Feature:Builds] result image should have proper labels set  S2I build from a template should create a image from test-s2i-build.json template with proper Docker labels [Suite:openshift/conformance/parallel]"\
```
Comment (r4f4) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:889]: Extraneous `\` at the end of the line. And you removed the quotes around test-s2i-build.json when you were supposed to escape them: \"test-s2i-build.json\"

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 6]
```diff
@@ -996,7 +1205,7 @@ periodics:
       name: job-definition
     - name: pull-secret
       secret:
-        secretName: regcred
+         secretName: regcred
```
Comment (r4f4) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:1208]: unrelated change

**Review by r4f4 — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 7]
```diff
@@ -755,6 +755,215 @@ periodics:
...
+          "[sig-builds][Feature:Builds] build have source revision metadata  started build should contain source revision information [Suite:openshift/conformance/parallel]"
+          "[sig-builds][Feature:Builds] build with empty source  started build should build even with an empty source in build config [Suite:openshift/conformance/parallel]"
+          "[sig-builds][Feature:Builds] build without output image  building from templates should create an image from a S2i template without an output image reference defined [Suite:openshift/conformance/parallel]"
+          "[sig-builds][Feature:Builds] custom build with buildah  being created from new-build should complete build with custom builder image [Suite:openshift/conformance/parallel]"
+          "[sig-builds][Feature:Builds] result image should have proper labels set  S2I build from a template should create a image from 'test-s2i-build.json' template with proper Docker labels [Suite:openshift/conformance/parallel]"
```
Comment (r4f4) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:889]: You can see from the test output that this test is not excluded because the name here doesn't match the name of the test:
```
[sig-builds][Feature:Builds] result image should have proper labels set S2I build from a template should create a image from "test-s2i-build.json" template with proper Docker labels [Suite:openshift/conformance/parallel]
```

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 8]
```diff
@@ -663,7 +663,7 @@ periodics:
         set -euo pipefail
 
         # fetch binaries and configure working env, prow doesn't allow init containers or a second container
-        dir=$(ARTIFACTS)/ostree
+        dir=$(ARTIFACTS)/ostree
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:666]: no extra space

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 9]
```diff
@@ -881,6 +881,215 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cluster: api.ci
+  cron: 0 4,16 * * *
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:886]: This will be the cron line for the other two tests as well.

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 10]
```diff
@@ -881,6 +881,215 @@ periodics:
...
+      - name: JOB_NAME_SAFE
+        value: e2e-remote-libvirt
+      - name: MULTIARCH_RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp-s390x/release-s390x:4.7
+      - name: RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp/release:4.7
+      - name: REMOTE_LIBVIRT_HOSTNAME
+        value: ocponz
+      - name: REMOTE_LIBVIRT_HOSTNAME_1
+        value: ocponz2
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:965]: All of these lines will need to override the env configuration in the tests below as well, since the "env" is different between power and Z the and example I showed you was for power.

Comment (jaypoulz) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:965]: The exception is the TEST_COMMAND, since that is specifying what test should be run. So that will be the same as in the power example.

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 11]
```diff
@@ -7434,3 +7643,541 @@ periodics:
...
+            name: cluster-secrets-metal
+    - configMap:
+        name: prow-job-cluster-launch-installer-remote-libvirt-e2e
+      name: job-definition
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: release-pull-secret
+      secret:
+        secretName: ci-pull-credentials
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:7781]: This whole config looks great!

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 12]
```diff
@@ -7434,3 +7643,541 @@ periodics:
...
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:7913]: This looks like an extra copy paste of the 4.5 example. You can delete this whole block.

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 13]
```diff
@@ -7434,3 +7643,541 @@ periodics:
...
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:8049]: This looks great! This is the last job you needed. You don't need to commit the image ecosystem jobs for ppc64le. :)

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 14]
```diff
@@ -7434,3 +7643,541 @@ periodics:
...
+        
+
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:8183]: I would go ahead a remove this block. @vijay-vin and @Basavaraju-G should be able to handle a power-specific PR. The reason for decoupling the two is to make sure they can land as quickly as possible without blocking each other.

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml (hunk 12 above)
Comment (jaypoulz) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:7913]: Even if you were to add the power job config, you'd want to update line https://github.com/openshift/release/pull/13269/files#diff-f20da9fe69faf7e3bfa2be8f95367c6ef26e92e246cad6c78424807b153cf18fR7790 since it still says 4.5. Please remove the block for now so our power partners can cover the extra logic on their side.

**Review by jaypoulz — CHANGES_REQUESTED** [review]
Overall this is 90% of the way there. Good work! Let me know if you have questions.

**Comment by jaypoulz** [comment]
Don't forget to update the allowlist:
https://github.com/openshift/release/blob/master/core-services/template-deprecation/_allowlist.yaml#L1924-L1980

You'll need to add:
```
          release-openshift-origin-installer-e2e-remote-libvirt-image-ecosystem-s390x-4.7:
            generated: false
            kind: release
```
```
          release-openshift-origin-installer-e2e-remote-libvirt-jenkins-e2e-s390x-4.7:
            generated: false
            kind: release
```
```
          release-openshift-origin-installer-e2e-remote-libvirt-s390x-4.7:
            generated: false
            kind: release
```
After their corresponding 4.6 entries.

**Comment by jaypoulz** [comment]
Also check these lines for extra spaces since you made the linter mad. :)

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/13269/pull-ci-openshift-release-yamllint/1332203435558178816

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml (hunk 10 above)
Comment (adevir91) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:965]: May i know how to differniate two types of tests i.e power and Z.

Comment (r4f4) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:965]: Power -> `ARCH == ppc64le`
Z -> `ARCH == s390x`

Comment (adevir91) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:965]: May i know where i need to override above configaration.

Comment (r4f4) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:965]: In the `env:` section.

Comment (adevir91) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:965]: i.e in all sections  that comes below the above section

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 15]
```diff
@@ -881,6 +881,215 @@ periodics:
...
+               "[sig-network][Feature:Router] The HAProxy router should override the route host with a custom value [Suite:openshift/conformance/parallel]"
+               "[sig-network][Feature:Router] The HAProxy router should run even if it has no access to update status [Suite:openshift/conformance/parallel]"
+               "[sig-network][Feature:Router] The HAProxy router should serve a route that points to two services and respect weights [Suite:openshift/conformance/parallel]"
+               "[sig-network][Feature:Router] The HAProxy router should serve routes that were created from an ingress [Suite:openshift/conformance/parallel]"
+               "[sig-network][Feature:Router] The HAProxy router should serve the correct routes when scoped to a single namespace and label set [Suite:openshift/conformance/parallel]"
+               "[sig-operator] an end user can use OLM can subscribe to the operator [Suite:openshift/conformance/parallel]"
+               "[sig-storage] Dynamic Provisioning [k8s.io] GlusterDynamicProvisioner should create and delete persistent volumes [fast] [Skipped:ibmcloud] [Suite:openshift/conformance/parallel] [Suite:k8s]"
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:1037]: Threre are trailing spaces on this line mentioned by the linter.
```
   1037:208  error    trailing spaces  (trailing-spaces) 
```

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 16]
```diff
@@ -7434,3 +7643,409 @@ periodics:
...
+        name: prow-job-cluster-launch-installer-remote-libvirt-e2e
+      name: job-definition
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: release-pull-secret
+      secret:
+        secretName: ci-pull-credentials
+        
+
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:8051]: Deleting these two lines will fix the other linter issues.
```
  8050:1    error    trailing spaces  (trailing-spaces)
  8051:1    error    too many blank lines (1 > 0)  (empty-lines) 
```

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 17]
```diff
@@ -881,6 +881,215 @@ periodics:
...
+                cpu: 100m
+                memory: 200Mi
+          tag_specification:
+            name: "$(BRANCH)"
+            namespace: ocp
+          tests:
+          - as: e2e-$(CLUSTER_TYPE)-all
+            commands: $(TEST_COMMAND)
+            openshift_installer:
+              cluster_profile:"$(CLUSTER_TYPE)"
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:955]: You're missing a space between cluster_profile: and "$(CLUSTER_TYPE)"

Like so: `cluster_profile: "$(CLUSTER_TYPE)"`

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 18]
```diff
@@ -881,6 +881,215 @@ periodics:
...
+               "[sig-network][Feature:Router] The HAProxy router should serve a route that points to two services and respect weights [Suite:openshift/conformance/parallel]"
+               "[sig-network][Feature:Router] The HAProxy router should serve routes that were created from an ingress [Suite:openshift/conformance/parallel]"
+               "[sig-network][Feature:Router] The HAProxy router should serve the correct routes when scoped to a single namespace and label set [Suite:openshift/conformance/parallel]"
+               "[sig-operator] an end user can use OLM can subscribe to the operator [Suite:openshift/conformance/parallel]"
+               "[sig-storage] Dynamic Provisioning [k8s.io] GlusterDynamicProvisioner should create and delete persistent volumes [fast] [Skipped:ibmcloud] [Suite:openshift/conformance/parallel] [Suite:k8s]"  
+            EOEXCLUDE
+            export PATH=/usr/libexec/origin:$PATH
+            openshift-tests run openshift/conformance/parallel --dry-run | ./invert_excluded.py excluded_tests > test-suite.txt
+            TEST_SUITE="--file=test-suite.txt" run-tests
```
Comment (jaypoulz) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:1041]: The indentation in this section is **really messed up**. This is what is giving you grief in you you config validations.
Here's how the block should be formatted. I've taken the liberty to remove the additional space while I was at it:
```
      - name: TEST_COMMAND
        value: |
          # these tests are not able to run in CI due to
          # missing test image dependencies.  They must be
          # run manually
          cat > invert_excluded.py <<EOSCRIPT
          #!/usr/libexec/platform-python
          import sys
          all_tests = set()
          excluded_tests = set()
          for l in sys.stdin.readlines():
            all_tests.add(l.strip())
          with open(sys.argv[1], "r") as f:
            for l in f.readlines():
              excluded_tests.add(l.strip())
          test_suite = all_tests - excluded_tests
          for t in test_suite:
            print(t)
          EOSCRIPT
          chmod +x invert_excluded.py
          cat > excluded_tests <<EOEXCLUDE
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] listing mutating webhooks should work [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] listing validating webhooks should work [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] patching/updating a mutating webhook should work [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should be able to deny attaching pod [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should be able to deny custom resource creation, update and deletion [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should deny crd creation [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should honor timeout [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should mutate configmap [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should mutate custom resource [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should mutate custom resource with different stored version [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should mutate custom resource with pruning [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should mutate pod and apply defaults after mutation [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should not be able to mutate or prevent deletion of webhook configuration objects [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] AdmissionWebhook [Privileged:ClusterAdmin] should unconditionally reject operations on fail closed webhook [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] CustomResourceConversionWebhook [Privileged:ClusterAdmin] should be able to convert a non homogeneous list of CRs [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-api-machinery] CustomResourceConversionWebhook [Privileged:ClusterAdmin] should be able to convert from CR v1 to CR v2 [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-apps][Feature:DeploymentConfig] deploymentconfigs when tagging images should successfully tag the deployed image [Suite:openshift/conformance/parallel]"
            "[sig-auth][Feature:LDAP] LDAP IDP should authenticate against an ldap server [Suite:openshift/conformance/parallel]"
            "[sig-auth][Feature:LDAP] LDAP should start an OpenLDAP test server [Suite:openshift/conformance/parallel]"
            "[sig-auth][Feature:ProjectAPI]  TestProjectWatch should succeed [Suite:openshift/conformance/parallel]"
            "[sig-auth][Feature:ProjectAPI]  TestScopedProjectAccess should succeed [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds] Multi-stage image builds  should succeed [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds] Optimized image builds  should succeed [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds] build can reference a cluster service with a build being created from new-build should be able to run a build that references a cluster service [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds] build have source revision metadata started build should contain source revision information [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds] build with empty source  started build should build even with an empty source in build config [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds] build without output image  building from templates should create an image from a S2i template without an output image reference defined [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds] custom build with buildah  being created from new-build should complete build with custom builder image [Suite:openshift/conformance/parallel]"           "[sig-builds][Feature:Builds] result image should have proper labels set S2I build from a template should create a image from \"test-s2i-build.json\" template with proper Docker labels [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds] s2i build with a quota  Building from a template should create an s2i build with a quota and run it [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds] s2i build with a root user image should create a root build and pass with a privileged SCC [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds][timing] capture build stages and durations should record build stages and durations for s2i [Suite:openshift/conformance/parallel]"
            "[sig-builds][Feature:Builds][valueFrom] process valueFrom in build strategy environment variables should successfully resolve valueFrom in s2i build environment variables [Suite:openshift/conformance/parallel]"
            "[sig-cli] Kubectl client Kubectl server-side dry-run should check if kubectl can dry-run update Pods [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
            "[sig-cli] Kubectl client Simple pod should return command exit codes [Suite:openshift/conformance/parallel] [Suite:k8s]"
            "[sig-cli] oc adm must-gather runs successfully [Suite:openshift/conformance/parallel]"
            "[sig-cli] oc adm new-project [Suite:openshift/conformance/parallel]"
            "[sig-cli] oc rsh rsh specific flags should work well when access to a remote shell [Suite:openshift/conformance/parallel]"
            "[sig-devex][Feature:Templates] templateinstance readiness test  should report ready soon after all annotated objects are ready [Suite:openshift/conformance/parallel]"
            "[sig-instrumentation] Prometheus when installed on the cluster shouldn't report any alerts in firing state apart from Watchdog and AlertmanagerReceiversNotConfigured [Early] [Suite:openshift/conformance/parallel]"
            "[sig-network] Networking should provide Internet connection for containers [Feature:Networking-IPv4] [Skipped:azure] [Suite:openshift/conformance/parallel] [Suite:k8s]"
            "[sig-network] multicast when using one of the plugins 'redhat/openshift-ovs-multitenant, redhat/openshift-ovs-networkpolicy' should allow multicast traffic in namespaces where it is enabled [Suite:openshift/conformance/parallel]"
            "[sig-network][Feature:Router] The HAProxy router should expose prometheus metrics for a route [Suite:openshift/conformance/parallel]"
            "[sig-network][Feature:Router] The HAProxy router should override the route host for overridden domains with a custom value [Suite:openshift/conformance/parallel]"
            "[sig-network][Feature:Router] The HAProxy router should override the route host with a custom value [Suite:openshift/conformance/parallel]"
            "[sig-network][Feature:Router] The HAProxy router should run even if it has no access to update status [Suite:openshift/conformance/parallel]"
            "[sig-network][Feature:Router] The HAProxy router should serve a route that points to two services and respect weights [Suite:openshift/conformance/parallel]"
            "[sig-network][Feature:Router] The HAProxy router should serve routes that were created from an ingress [Suite:openshift/conformance/parallel]"
            "[sig-network][Feature:Router] The HAProxy router should serve the correct routes when scoped to a single namespace and label set [Suite:openshift/conformance/parallel]"
            "[sig-operator] an end user can use OLM can subscribe to the operator [Suite:openshift/conformance/parallel]"
            "[sig-storage] Dynamic Provisioning [k8s.io] GlusterDynamicProvisioner should create and delete persistent volumes [fast] [Skipped:ibmcloud] [Suite:openshift/conformance/parallel] [Suite:k8s]"
          EOEXCLUDE
          export PATH=/usr/libexec/origin:$PATH
          openshift-tests run openshift/conformance/parallel --dry-run | ./invert_excluded.py excluded_tests > test-suite.txt
          TEST_SUITE="--file=test-suite.txt" run-tests
```

**Review by jaypoulz — CHANGES_REQUESTED** [review]
There other issues are actually pretty obvious if you load up the comparison of the blocks in meld:
https://meldmerge.org/

Meld is one of my favorite comparison tools. What i did was paste the 4.6 block on one side, and 4.7 on the other, and it highlights the differences

Your issues are with indentation. Indentation and attention to details is **crucial** when you're working both with yaml and python. Yaml uses indentation to understand object structure, and python uses it for control flow. So you have to get them exactly right every time.

I will send you an exact copy of the 4.6 block, your 4.7 block, and my 4.7 block for you to load into meld so you can see the differences highlighted.

**Comment by jaypoulz** [comment]
One last thing I want you to update as part of this PR.
Please add a line under the one I added in: https://github.com/openshift/release/pull/13996

Instead of ppc64le, it should be s390x.
This will ensure that your test results are available in test grid, which means we'll see them in the sippy dashboard.
